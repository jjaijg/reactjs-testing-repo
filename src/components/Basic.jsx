import PropTypes from 'prop-types';

const Basic = (props) => {
  return <div>Basic component</div>;
};

Basic.propTypes = {};

export default Basic;
