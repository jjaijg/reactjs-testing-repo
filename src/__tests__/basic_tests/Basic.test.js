import Basic from '../../components/Basic';

import Enzyme, { shallow, render, mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';

Enzyme.configure({
  adapter: new Adapter(),
});

it('Render Enzyme correctly', () => {
  const wrapper = shallow(<Basic />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
