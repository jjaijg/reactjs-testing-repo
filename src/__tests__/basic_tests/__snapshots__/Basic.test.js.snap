// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Enzyme correctly 1`] = `
<div>
  Basic component
</div>
`;
